src/Raven.Client/Documents/Identity/AsyncHiLoIdGenerator.cs

```diff
@@ -67,15 +73,12 @@ public RangeValue(long min, long max)
             }
         }
 
-        private Lazy<Task> _nextRangeTask = new Lazy<Task>(() => Task.CompletedTask);
-        protected string ServerTag;
-
         /// <summary>
         /// Generates the document ID.
         /// </summary>
         /// <param name="entity">The entity.</param>
         /// <returns></returns>
-        public async Task<string> GenerateDocumentIdAsync(object entity)
+        public async Task<string> GenerateDocumentIdAsync()
```

Comment:
You are changing the public API here, why?

Reply:
`entity` was not used
reverted this